### PR TITLE
 Adding option `--output_filepath` for tflite benchmark

### PR DESCRIPTION
**Pull request by zotanika**
- Exporting output tensors to a file in raw data format.
- Supposed to be helpful for automating on-device(e.g. android-arm64 target) runtime validation of tflite models.

### CODE REVIEW

**Comment by zotanika** [comment]
> @zotanika Can you please resolve conflicts? Thank you!



> @zotanika Can you please resolve conflicts? Thank you!
Thank you for the note. Conflicts have been fixed, and waiting for completion of CI check.

**Comment by gbaned** [comment]
Hi @terryheo Can you please review this PR ? Thank you!

File: tensorflow/lite/tools/benchmark/benchmark_tflite_model.cc [hunk 1]
```diff
@@ -378,7 +407,10 @@ std::vector<Flag> BenchmarkTfLiteModel::GetFlags() {
                        "are not used."),
       CreateFlag<int32_t>(
           "optimize_memory_for_large_tensors", &params_,
-          "Optimize memory usage for large tensors with sacrificing latency.")};
+          "Optimize memory usage for large tensors with sacrificing latency."),
+      CreateFlag<std::string>(
+          "save_outputs_in_file", &params_,
```
Comment (terryheo) [inline tensorflow/lite/tools/benchmark/benchmark_tflite_model.cc:412]: How about "output_filepath" ?

**Review by terryheo — COMMENTED** [review]
Thanks for your contribution!

File: tensorflow/lite/tools/benchmark/benchmark_tflite_model.cc (hunk 1 above)
Comment (zotanika) [reply tensorflow/lite/tools/benchmark/benchmark_tflite_model.cc]: > 

Thanks for suggesting the simpler option. I've picked it up.

**Review by terryheo — APPROVED** [review]
LGTM

**Pull request merged**